services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -501,6 +499,35 @@ async def _discover_running_services(self) -> None:
         for scheduler_data in services_to_observe:
             await self.add_service(scheduler_data)
 
+    async def _cleanup_volume_removal_services(self) -> None:
+        settings: DynamicServicesSchedulerSettings = (
+            self.app.state.settings.DYNAMIC_SERVICES.DYNAMIC_SCHEDULER
+        )
+        dynamic_sidecar_settings: DynamicSidecarSettings = (
+            self.app.state.settings.DYNAMIC_SERVICES.DYNAMIC_SIDECAR
+        )
+
+        logger.debug(
+            "dynamic-sidecars cleanup pending volume removal services every %s seconds",
+            settings.DIRECTOR_V2_DYNAMIC_SCHEDULER_PENDING_VOLUME_REMOVAL_S,
+        )
+        while True:
+            logger.debug("Removing pending volume removal services...")
+
+            try:
+                await remove_pending_volume_removal_services(dynamic_sidecar_settings)
+            except asyncio.CancelledError:  # pragma: no cover
```

Comment:
what is the #pragma: no cover for?

Reply:
My bad not required here. It's used to tell code coverage to ignore this line when computing coverage.